dev-lang/vala/vala-0.36.11.recipe

```diff
@@ -64,6 +73,20 @@ BUILD_PREREQUIRES="
 	cmd:pkg_config$secondaryArchSuffix
 	"
 
+defineDebugInfoPackage vala$secondaryArchSuffix \
+	$libDir/libvala-${portVersion%.*}.so.0.0.0 \
+	$binDir/vala \
+	$binDir/valac \
+	$binDir/vapicheck \
+	$binDir/vapigen \
+	$binDir/vala-${portVersion%.*} \
+	$binDir/valac-${portVersion%.*} \
+	$binDir/vapicheck-${portVersion%.*} \
+	$binDir/vapigen-${portVersion%.*}
+	# looks like it doesn't take more then one "-" here
+	#$binDir/vala-gen-introspect \
+	#$binDir/vala-gen-introspect-${portVersion%.*} \
+
 BUILD()
```

Comment:
I'd change this in `TEST()`:
```diff
 TEST()
 {
- 	sed -i -e 's/-lm/-lroot/g' $sourceDir/tests/testrunner.sh
 	make check
 }
```
and insert a `PATCH()` before `BUILD()`:
```
PATCH()
{
 	sed -i -e 's/-lm/-lroot/' tests/testrunner.sh
}
```
(I dropped the `g` and the `$sourceDir/`)